packages/react-core/src/components/Form/examples/Form.md

```diff
@@ -420,6 +598,38 @@ class HorizontalForm extends React.Component {
 }
 ```
 
+### Horizontal helper text on top
+```js
+import React from 'react';
+import { Form, FormGroup, Checkbox } from '@patternfly/react-core';
+
+class HorizontalFormHelperTextOnTop extends React.Component {
+  
+  constructor(props) {
+    super(props);
+    this.state = {
+      helperText: 'Select all that apply'
+    };
+  }
+
+  
+  render() {
+    return (
+      <Form isHorizontal>
+        <FormGroup 
+          label="Label" 
+          helperText={this.state.helperText}
+          isHelperTextBeforeField 
+          fieldId="options">
```

Comment:
Can you add `hasNoPaddingTop`? We use that when the thing in the form element field isn't a text input, textarea, or select.